src/test/java/com/facebook/openwifirrm/modules/ApiServerTest.java

```diff
@@ -507,9 +507,29 @@ void testCORS() throws Exception {
 	@Test
 	@Order(2000)
 	void test_system() throws Exception {
-		HttpResponse<JsonNode> resp = Unirest.get(endpoint("/api/v1/system?command=info")).asJson();
-		assertEquals(200, resp.getStatus());
-		assertEquals(VersionProvider.get(), resp.getBody().getObject().getString("version"));
+		// Test on GET api
+		HttpResponse<JsonNode> get_resp = Unirest.get(endpoint("/api/v1/system?command=info")).asJson();
+		assertEquals(200, get_resp.getStatus());
+		assertEquals(VersionProvider.get(), get_resp.getBody().getObject().getString("version"));
+
+		// Test on POST api
+		String url = endpoint("/api/v1/system");
+		// Valid command
+		JSONObject json_obj = new JSONObject("{\"command\": \"reload\"}");
```

Comment:
To clarify, you *don't* need to create a `JSONObject` here (or the other ones below). Just pass it as a string literal in the function call.